### PR TITLE
global: package style and initialization

### PR DESCRIPTION
**Pull request by jirikuncar**
- adds API autodoc (addresses #57)

### CODE REVIEW

File: invenio_oauth2server/forms.py [hunk 1]
```diff
@@ -61,11 +61,13 @@ def scopes_multi_checkbox(field, **kwargs):
...
-        html.append(u'<label for="%s" class="checkbox-inline">' % field_id)
-        html.append(u'<input %s /> ' % widgets.html_params(**options))
-        html.append("%s <br/><small class='text-muted'>%s</small>" % (
-            value, label.help_text)
-        )
+        html.append(u'<label for="{0}" class="checkbox-inline">'.format(
+            field_id
+        ))
+        html.append(u'<input {0} /> '.format(widgets.html_params(**options)))
+        html.append('%s <br/><small class="text-muted">{0}</small>'.format(
```
Comment (slint) [inline invenio_oauth2server/forms.py]: Should this string also be `u` prefixed?

Comment (jirikuncar) [inline invenio_oauth2server/forms.py]: I would say, that we should remove the `u` not to mix it. IMHO it's coming from Invenio 2.x times when we had troubles with unicode in Jinja templates.

Comment (slint) [inline invenio_oauth2server/forms.py]: Although in Python 2.7:
``` python
In [1]: print('English: {0}, Greek: {1}'.format('lambda', u'λάμδα'))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-4: ordinal not in range(128)

In [2]: print(u'English: {0}, Greek: {1}'.format('lambda', u'λάμδα'))
English: lambda, Greek: λάμδα
```

And `field.iter_choices()` could very easily include a unicode string. Also in [the WTForms docs](http://wtforms.readthedocs.io/en/latest/widgets.html#custom-widgets) they also have this snippet using the `u` prefix.

Comment (jirikuncar) [inline invenio_oauth2server/forms.py]: Ok then. I have fixed the one missing `u` prefix.

**Comment by lnielsen** [comment]
LGTM

**Pull request merged**